tracer/src/Datadog.Trace/AppSec/AddressesConstants.cs

```diff
@@ -21,12 +21,13 @@ internal static class AddressesConstants
         public const string RequestPathParams = "server.request.path_params";
         public const string RequestBodyCombinedFileSize = "server.request.body.combined_file_size";
         public const string RequestClientIp = "http.client_ip";
-        public const string ResponseStatus = "server.response.status";
 
+        public const string ResponseStatus = "server.response.status";
         public const string ResponseBodyRaw = "server.response.body.raw";
```

Comment:
Minor: I think this address is unused, and never will be, since we're not interested in analyzing raw values, so could be removed.

Reply:
thanks removed in [2c3e128](https://github.com/DataDog/dd-trace-dotnet/pull/4654/commits/2c3e128c69e41e88729ec846063f3e83461be5d2)